Formula/qt5.rb

```diff
@@ -86,6 +87,7 @@ class Qt5 < Formula
   depends_on :mysql => :optional
   depends_on :postgresql => :optional
   depends_on :xcode => :build
+  depends_on "openssl" => :optional
```

Comment:
Was just griping about OpenSSL potentially being the default for Qt5, heh.

Secure Transport is friendlier, and I think you'd cause yourself issues overriding the upstream default to use OpenSSL given people have been trying to get Homebrew to do that to `curl` for a while _(where the upstream default is actually OpenSSL)_ and we kept shooting it down because SecureTransport is shipped by the system and decently secure/maintained/etc.